arabic/learn/learn.md

```diff
@@ -1,3 +1,4 @@
 اذا قررت ان تكمل الالتعلم فانه عليك ان تبحث عن موضوع مكمل لموضوع السياق الماضي، هل تعرف لماذا؟
 لانه يجب عليك الاستفاده من السياق الماضي وتطوير قدراتك فيه لذا ماذا تتوقع ان تفعل الان؟
 قال صلى الله عليه وسلم من سلك طريقاً يلتمس فيه علماً سهل الله له طريقاً الى الجنة
+[و لماذا التعلم ](learn\whylearning\speculate.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.